infra/aws/terraform/prow-build-cluster/resources/monitoring/README.md

```diff
@@ -0,0 +1,51 @@
+# EKS monitoring
+
+## Setting up monitoring
+
+```bash
+# Create monitoring namespace:
+kubectl apply -f ../namespaces.yaml
+
+# Install CRDs for Prometheus Operator:
+# (server side is required due to long annotations)
+kubectl apply --server-side -f ./prometheus-operator-crds
+
+# Install Prometheus Operator:
+kubectl apply  -f ./prometheus-operator
+
+# Install kube-state-metrics
+kubectl apply -f ./kube-state-metrics
+
+# Install node-exporter
+kubectl apply -f ./node-exporter
+
+# Install cadvisor
+kubectl apply -f ./cadvisor
+
+# Install dashboards for Grafana
+kubectl apply --server-side -f ./grafana/dashboards
+
+# Install Grafana
+kubectl apply -f ./grafana
+```
+
+[Prometheus Operator CRDs](https://github.com/prometheus-operator/prometheus-operator/tree/v0.63.0/example/prometheus-operator-crd-full)
+
+## Local access
+
+```bash
+# Prometheus
+kubectl --namespace monitoring port-forward svc/prometheus-operated 9090
+
+# Grafana
+kubectl --namespace monitoring port-forward svc/grafana 3000
```

Comment:
Should we expose this Grafana instance publicly? If yes, let's create a follow up issue to track that.

Reply:
I wasn't sure if this is something that we want to maintain in a longer term. IMO we can expose it for now till we haven't figured what monitoring solution works the best for both GCP and AWS clusters.

I wonder if there were other things we want to expose over time, in which case ingress-controller would be handy.